src/main/frontend/handler/file.cljs

```diff
@@ -121,7 +121,7 @@
                    (fn [_]
                      (cond
                        (= path (config/get-repo-config-path repo))
-                       (p/let [_ (repo-config-handler/restore-repo-config! repo)]
+                       (p/let [_ (repo-config-handler/restore-repo-config! repo content)]
```

Comment:
`content` is passed explicitly here because the `:file/content` value for the repo's config might be empty at this time.

Reply:
Thanks for fixing. I remember you changed `restore-global-config!` to read directly from the file to avoid timing issues with reading from the db. Wouldn't that be an ideal way to fix this so repo-config doesn't depend on db state at all? `logseq/config.edn` is updated correctly when we add favorites. Happy to make this change if it's preferred

Reply:
Good idea! That'll be better.